common/git-hooks/pre-push

```diff
@@ -58,6 +66,9 @@ function extractPathPrefix(inputString) {
   const match = inputString.match(regex);
 
   if (match && match[1]) {
+    if(match[1].endsWith("perf-tests")) {
```

Comment:
As the logic in this file has been evolving.. Have you considered moving this script to the dev-tool and adding some tests?